python/mlcroissant/mlcroissant/_src/structure_graph/nodes/file_object.py

```diff
@@ -22,39 +23,52 @@ class FileObject(Node):
 
     content_url: str | None = None
     content_size: str | None = None
-    contained_in: list[str] | None = None
+    contained_in: list[str | dict[str, str | None]] | None = None
```

Comment:
Rollback to `contained_in: list[str] | None = None`

Reply:
Shall I do that? Isn't the current signature `contained_in: list[str | dict[str, str | None]] | None` right?

Reply:
Done.